tests/TestCase.php

```diff
@@ -3,8 +3,37 @@
 namespace Tests;
 
 use Illuminate\Foundation\Testing\TestCase as BaseTestCase;
+use Illuminate\Support\Facades\Artisan;
 
 abstract class TestCase extends BaseTestCase
 {
     use CreatesApplication;
+    protected $user, $token;
+
+    /**
+     * Set up TestCase
+     *
+     * @return void
+     */
+    public function setUp()
+    {
+        parent::setUp();
+        Artisan::call('passport:install');
+
+        $this->user = factory('App\Models\User')->create();
+        $this->token =  $this->user->createToken('token')->accessToken;
```

Comment:
ko cần gán biến $token đâu em, em làm sao nó lưu đc vào $user, rồi mình gọi $this->user->token là đc